lib/msf/base/serializer/opt_conditions.rb

```diff
@@ -0,0 +1,54 @@
+# -*- coding: binary -*-
+module Msf
+  module Serializer
+    module OptConditions
+
+      # Check a condition's result
+      def self.eval_condition(left_value, operator, right_value)
+        result = true
+        case operator
+        when "=="
+          result = left_value == right_value
+        when "!="
+          result = left_value != right_value
+        when "in"
+          result = right_value.include?(left_value)
+        when "nin"
+          result = right_value.include?(left_value)
+          result = !result
+        end
+
+        result
+      end
+
+      # Check an OPTION conditions. This function supports
+      # dump_options()
+      def self.opt_condition_checked(condition, mod, opt)
+        result = true
+
+        if ((condition != []) and (condition != nil))
```

Comment:
Instead of using this and then indenting this whole block, you can change this to:

```
return true if condition.nil? || condition.empty?
```

If the condition isn't defined, it'll then default to showing the option.

Reply:
Done!